docs/source/cookbook/ccdc.rst

```diff
@@ -1,10 +1,475 @@
+
+.. warning::
+
+    For very cloud-prone regions, it is also possible to combine the Landsat data with the Sentinel-2 data to densify the underlying time-series. Note that because of differences in the sensors (although band names are equal) and the overpass time, artifacts may be introduced that will affect the breakpoint detection.
+
+The breakpoint detection is at the heart of CCDC. The respective selection of bands can considerably affect the outcome of the CCDC breakpoint detection. Unfortunately, there does not seem to be a *"one size fits all"* preset for all kinds of applications. Scientific evidence is suggesting to use all color bands but the blue `Zhu et al 2020 <https://www.sciencedirect.com/science/article/pii/S0034425719301002>`_. According to the study, the selection of additional ratio bands does not add any improvement. However, it should be noted that this assumption is based on the detection of all types of land cover changes and that the study uses a modified version of CCDC (named COLD), where the change in bands are weighted differently than in the original version used in SEPAL, respectively Google Earth Engine.
+
+.. tip::
+
+    The use of the color bands allows you to later select the *Green* and the *Swir1* band as TMASK bands for CCDC's internal, multi-temporal cloud removal. You find this in the :guilabel:`OPT` button pop-up menu under :guilabel:`MORE`.
+
+If the creation of the CCDC asset is aimed at the detection of both, forest degradation and deforestation, the *Normalized Degradation Forest Index* NDFI might be another suitable choice as applied by `Bullock et al 2020 <https://www.sciencedirect.com/science/article/pii/S0034425718305200>`_. Note that this article and the NDFI are specifically tested over tropical rainforest of the Brazilian Amazon. Changes in other forest types might be better captured by different ratios or the color bands. For instance one can consider the *Normalized Differenced Moisture Index* NDMI when looking at Mangrove forests.
+
+.. tip::
+    In case of doubt, we suggest to go for the default option and use all of the color bands, except the blue one.
+
+.. thumbnail:: ../_images/cookbook/ccdc_asset/sensor_selection_color_breakbands.png
+    :title: Sensor Selection - Color breakpoint bands
+    :width: 49%
+    :group: ccdc-asset-recipe
+
+.. thumbnail:: ../_images/cookbook/ccdc_asset/sensor_selection_ndfi_breakband.png
+    :title: Sensor Selection - NDFI breakpoint band
+    :width: 49%
+    :group: ccdc-asset-recipe
+
+Radar Data
+""""""""""
+
+In order to create a CCDC asset based on underlying radar time-series, you need to select the :guilabel:`RADAR` button. This will make use the Sentinel-1 C-Band SAR Image Collection in Google Earth Engine. To the best of our knowledge, no scientific studies have been done that investigate the ideal band selection for breakpoint detection. As a starting point we suggest to use the default option that includes the VV and the VH band.
```

Comment:
bold or italic for the satellite name ?

Reply:
hmm, I think I did not use throughout the fiel, but I'll recheck to keep it consistent. what do you use?

Reply:
I would go for italic